src/components/Loader.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Backdrop from "@material-ui/core/Backdrop";
+
+export default function Loader(props) {
+  return (
+    <div>
+      <Backdrop open={props.open || false}>
```

Comment:
you can do `open={!!props.open}` (no matter what it is it will cast it to boolean)